src/lib/utils/language.js

```diff
@@ -21,6 +21,25 @@ const languageNames = {
   fr: 'Français',
 };
 
+/**
+ * When we display a list of translations available for a given page,
+ * use this ordering for the names of the languages.
+ * See https://github.com/GoogleChrome/web.dev/issues/7430
+ * @const
+ */
+const languageOrdering = [
```

Comment:
I think this is a cleaner approach, rather than trying to group the languages into script families and then sort within the script families (as in @tannerdolby's original PR).

The ordering in this array reflects the ordering that we'd like the supported locale translations to appear in.

If we add more supported locales in the future, we'd place the two-letter language code in the appropriate sport in this array.

(If there's a locale that we end up supporting but we forget to add it to this array, the way the `sort()` works, it will just end up being sorted to the front of the list, so it fails-safe.)